src/engine/input/key-mapping/InputGroupItem.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using Godot;
+using Newtonsoft.Json;
+
+public class InputGroupItem : VBoxContainer
+{
+    [Export]
+    [JsonIgnore]
```

Comment:
Are json properties in this needed?

Reply:
those JsonIgnore properties were useless, because those were not props.
The JsonPropertyAttributes are redundant, but clarify where the data comes from.
Should I remove the unnecessary JsonPropertyAttributes?

Reply:
They should be removed. I'm very confused why you are using JSON serialization on the GUI nodes? And in the options menu of all places, that just seems *wrong* to me.